.travis.yml

```diff
@@ -0,0 +1,32 @@
+sudo: false
+
+language: python
+
+python:
+  - 2.7
+  - 3.2
+  - 3.3
+  - 3.4
+  - "pypy"
+  - "pypy3"
+
+cache:
+  directories:
+    - $HOME/.cache/pip
+
+addons:
+  apt:
+    sources:
+      - build-essential
+
+install:
+ - pip install --upgrade pip
+ - pip install --install-option="--no-cython-compile" Cython 
+ - pip install .
+
+script:
+ - cd `mktemp -d`
+ - nosetests -v nxlemon
```

Comment:
Well, in general practice and also as in networkx we should not run `nosetests` for each and every test module. This will inflate the file after every new module is created. nosetests is able to find them.

In case of `nxmetis` there's only one test module so no worries.